src/common/actionMenu.component.tsx

```diff
@@ -12,10 +12,14 @@ import { StyledUppyBox } from './uppy.utils';
 
 export interface ActionMenuProps {
   ariaLabelPrefix: string;
-  editMenuItem: { onClick: () => void; dialog: React.ReactNode };
+  editMenuItem: {
+    onClick: (isPrivilegedUser?: boolean) => void;
```

Comment:
Can this been change to `props?: {isPrivilegedUser?:boolean}`. As the code below in this file 

`editMenuItem.onClick(true); `

This is unclear and it would be more clearer if it is 

`editMenuItem.onClick({isPrivilegedUser:true});`